sources/packages/web/src/services/InstitutionService.ts

```diff
@@ -32,12 +32,12 @@ import {
   SearchInstitutionAPIOutDTO,
   InstitutionBasicAPIOutDTO,
   CreateInstitutionAPIInDTO,
-  InstitutionUserTypeAndRoleAPIOutDTO,
```

Comment:
As we removed all the referenced for the InstitutionUserTypeAndRoleAPIOutDTO, please remove them in the institution.dto
![image](https://user-images.githubusercontent.com/62901416/179099138-5ac860e7-d5a7-4d7a-b577-14b3cd1cf110.png)

Reply:
deja vu 😉

Reply:
Removed!